linera-views/src/dynamo_db.rs

```diff
@@ -1037,11 +1039,18 @@ impl KeyValueStoreClient for DynamoDbClientInternal {
     type Keys = DynamoDbKeys;
     type KeyValues = DynamoDbKeyValues;
 
+    fn max_key_size(&self) -> usize {
+        MAX_KEY_BYTES
+    }
+
     fn max_stream_queries(&self) -> usize {
         self.max_stream_queries
     }
 
     async fn read_key_bytes(&self, key: &[u8]) -> Result<Option<Vec<u8>>, DynamoDbContextError> {
```

Comment:
This method name is starting to hurt. It should probably be `read_value_bytes` or `read_value`.

Reply:
Ok, following renames were done:
* `read_key_bytes` => `read_value_bytes`
* `read_key` => `read_value`
* `read_multi_key_bytes` => `read_multi_values_bytes`
* `read_multi_key` => `read_multi_values`